tools/nni_cmd/nnictl.py

```diff
@@ -34,7 +34,10 @@
 
 def nni_info(*args):
     if args[0].version:
-        print(pkg_resources.get_distribution('nni').version)
+        try:
+            print(pkg_resources.get_distribution('nni').version)
+        except:
+            print_error('Get version failed, please use `pip3 list | grep nni` to check nni version!')
```

Comment:
can you just catch the expected exception?

Reply:
Yes, changed to catch exception.